main/modes/paint/paint_common.h

```diff
@@ -59,6 +59,7 @@
 
 #define PAINT_TOOLBAR_FONT "ibm_vga8.font"
 #define PAINT_SAVE_MENU_FONT "radiostars.font"
+#define PAINT_SMALL_FONT "tom_thumb.font"
```

Comment:
I haven't seen this in action yet, but I generally recommend against it. It was hard to read on the 128x64 screen last year, and its not gonna be any easier on the 280x240 with a smaller pixel pitch. It may be OK with a blown up emulator, but make sure to shrink the window for 1:1 pixels and see it there.